src/components/BetaInfoBox.jsx

```diff
@@ -0,0 +1,44 @@
+/* global _ */
+import React, { useState } from 'react';
+import Alert from './ui/Alert';
+import { Button } from './ui';
+import { getBetaPopupClosed, setBetaPopupClosed } from 'src/adapters/store';
+import { parseQueryString } from '../libs/url_utils';
+
+const initialQueryParams = parseQueryString(window.location.search);
+
+const BetaInfoBox = () => {
+  const [closed, setClosed] = useState(getBetaPopupClosed());
+  const [client] = useState(initialQueryParams['client']);
+
+  const closeBetaPopup = () => {
+    setBetaPopupClosed();
+    setClosed(true);
+  };
+
+  const footer = <div className="alert-link">
+    <Button variant="tertiary" onClick={closeBetaPopup}>
+      {_('Do not display this message again')}
+    </Button>
+  </div>;
+
+  return (
+    // Show beta popup if browser language is not french, if popup hasn't already been closed, and if the user comes from an IA
+    !navigator.language.includes('fr')
```

Comment:
For consistency, I would prefer to use `window.getLang()` which is already defined by the application

Reply:
Also, we should still avoid `includes` on arrays and strings until we drop the support for IE11.

BTW, I've opened an issue to track the things we'll be able to do then, feel free to suggest others I forgot :) https://github.com/QwantResearch/erdapfel/issues/978